README.md

```diff
@@ -0,0 +1,63 @@
+<h1>Food Haven</h1>
+
+- In a world with food deserts, Food Haven supplies the public with location services for local farmer's markets. The search for markets focuses on farmer's markets that accept Supplemental Nutrition Assistance Program (SNAP) benefits and identifies the type of benefit assistance at the market if the data is available. The user can then select a market they wish to visit and utilizing the maps portion of the application, the user may locate public service transit routes and schedules available to connect them to the market from their current location. 
+
+<b>Developers:</b>
+
+Kailey Kaes: https://github.com/kaileykaes
+
+Jesse Sorman: https://github.com/Jesse193
+
+Kim Bergstrom: https://github.com/kbergstrom78
+
+Andy Stilmock: https://github.com/AStilmock
+
+<h2>FE: https://github.com/FoodHaven/FE_APP</h2>
+
+<h3>For Developers:</h3>
+
+Dependencies: 
+
+Microservice: https://github.com/FoodHaven/microservice-api
+
+Back end: https://github.com/FoodHaven/BE_APP
+
+<h3>Getting started</h3>
+
+Rails Version: 7.0.6
+
+- Run `bundle install` to get all Gems to work properly
+
+- Run `rails db:{drop,create,migrate,seed}`
+
+- Google Oauth 2.0
+  - Food Haven features Google Outh as an option to login. When using locally a OAuth client ID and client secret are needed for this to function correctly.
```

Comment:
What about this for more clarity:
When running this program on a local server, users will need OAuth client ID and client secret for this feature to function correctly.